src/components/connected/Connected.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { connect } from 'redux-bundler-react'
+import { translate } from 'react-i18next'
+
+const pickColor = (ipfsReady, ipfsConnected) => {
+  if (ipfsReady && ipfsConnected) return 'aqua'
+  if (ipfsReady) return 'yellow'
+  return 'gray'
+}
+
+const pickTitleKey = (ipfsReady, ipfsConnected) => {
+  if (ipfsReady && ipfsConnected) return 'connectedToIpfs'
+  if (ipfsReady) return 'ipfsApiRequestFailed'
```

Comment:
Shouldn't we put these keys into some constants file and import them from there?

Reply:
maybeeeee. I'm going to leave that out for now, as in general, different parts of the app should uses different i18n resource file. The crime here is that I'm reusing the same label in two, only mildly related things, when really each should use it's own resource file.